Tests/RoslynPluginGeneratorTests/DiagnosticAssemblyScannerTests.cs

```diff
@@ -20,110 +21,119 @@ public class DiagnosticAssemblyScannerTests
         public TestContext TestContext { get; set; }
 
         [TestMethod]
-        [DeploymentItem("ExampleAnalyzer1.dll", "./oneAnalyzer")]
-        public void ScanDirectory_OneAnalyzerAssembly()
+        public void InstantiateDiags_CSharp_NoFiles()
```

Comment:
Tests modified since the scanner no longer searches directories: it's handed a list of known files.
There are strong similarities between the old and new tests, but it's probably easier to review the new test code on its own.